lib/vicadmin/validate.go

```diff
@@ -172,20 +174,42 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 
 	// Network Connection Check
 	hosts := []string{
-		"google.com:80",
-		"docker.io:443",
+		"http://google.com",
+		"https://docker.io",
 	}
+
 	nwErrors := []error{}
 
-	for _, host := range hosts {
-		conn, err := net.DialTimeout("tcp", host, time.Minute/2)
+	// create a http client with a custom transport. The transport uses the [s/h]proxy defined in vicmachine create if applicable.
+	client := &http.Client{Timeout: 30 * time.Second}
+	vchArgs := make(map[string]string)
+	for _, args := range vch.VicMachineCreateOptions {
```

Comment:
I don't think what I changes should alter guestinfo, just access it. 
Even if I changed the Session Client to ignore the http variable there still may be other places that use http.Client in vicadmin or it's vendored dependencies. 
Basically, it seems like the proper approach to only include the proxy where needed instead of excluding it in all the place it's not needed.